core/src/main/java/com/linecorp/armeria/server/logging/ContentPreviewingService.java

```diff
@@ -140,6 +140,20 @@ public static ContentPreviewingServiceBuilder builder(ContentPreviewerFactory co
         this.responsePreviewSanitizer = responsePreviewSanitizer;
     }
 
+    /**
+     * Returns the specified {@link ContentPreviewerFactory}.
+     */
+    public ContentPreviewerFactory contentPreviewerFactory() {
+        return contentPreviewerFactory;
+    }
+
+    /**
+     * Returns the specified {@link BiFunction} that sanitizes the request content preview.
+     */
+    public BiFunction<? super RequestContext, String, ? extends @Nullable Object> responsePreviewSanitizer() {
```

Comment:
`super` and `extends` are useful when they are applied to method arguments.
```suggestion
    @UnstableApi
    public BiFunction<RequestContext, String, ? extends @Nullable Object> responsePreviewSanitizer() {
```

Reply:
It is not a big deal. Let's keep it as is.